docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: "3.8"
+
+services:
+  ollama:
+    build: .
+    container_name: ollama
+    image: ollama/ollama
```

Comment:
build and image ?

Reply:
Thanks for the comment. The reason they're both here:

- `build`: connect to the `Dockerfile` in this directory so can run `docker compose build`
- `image`: enable `docker compose up` to (1) pull, and if not found in registry, then (2) build

[This answer](https://stackoverflow.com/questions/34316047/difference-between-image-and-build-within-docker-compose/) shares a variety of perspectives on this